core/src/main/java/de/flo56958/MineTinker/Listeners/ItemListener.java

```diff
@@ -171,4 +172,19 @@ public void onItemBreak(PlayerItemBreakEvent event) {
 			}
 		} // no else as it gets added in if-clause
 	}
+
+	@EventHandler
+	public void onItemUse(PlayerItemDamageEvent event) {
+		ItemStack item = event.getItem();
+
+		if (modManager.isToolViable(item) && modManager.isArmorViable(item)) {
```

Comment:
you also need to check modManager.isWandViable(item) as Builderswands should not be converted on use

Reply:
Probably fixed.